lib/clientv3/ippool_e2e_test.go

```diff
@@ -831,8 +831,13 @@ var _ = testutils.E2eDatastoreDescribe("IPPool tests (etcd only)", testutils.Dat
 			}, options.SetOptions{})
 			Expect(err).NotTo(HaveOccurred())
 
+			// Create test node
```

Comment:
See this test: https://github.com/projectcalico/libcalico-go/blob/82d36d69d1fc0c6317b261c58bdcb7948766723f/lib/clientv3/ippool_e2e_test.go#L167

I think we should enhance it so that `spec1_2` includes a nodeselector.

The test includes an assertion that you can write to datastore, read it back, and the values are the same.

Reply:
addressed in 9fe3116